paws_collector.js

```diff
@@ -479,7 +491,7 @@ class PawsCollector extends AlAwsCollector {
             ],
             Namespace: 'PawsCollectors'
         };
-        let errorCode = typeof (error) === 'object' && error.errorCode ? error.errorCode : 'unknown';
+        let errorCode = typeof (error) === 'object' && error.errorCode ? error.errorCode : `unknown_${this.collector_id}`;
```

Comment:
@kkuzmin what do you think of adding the collector id to the "unknown" error tag? Are we going to get too many tags? I think it should be ok.

Reply:
I've been advised against doing this in the past. Typically you want to keep instance specific dimensions out of datadog metrics, apparently due to cost issues, though maybe this has changed. 

I'll reach out to cloud ops team about it.

Reply:
I haven't heard anything changed about tags. I advise to not do this.
If required we should always be able to find collector ID in service logs in kibana

Reply:
Sounds reasonable.

The one issue with these client errors is that they never reach our back end. So if we see client errors in DD, we don't really have a good way of finding out which collector is causing the error.

crazy idea. could we put all the cloudwatch logs for paws collectors in search?